src/commands/publish/mod.rs

```diff
@@ -107,21 +97,50 @@ fn publish_script(
         failure::bail!(error_msg(res_status, res_text))
     }
 
-    let pattern = if target.route.is_some() {
-        log::info!("publishing to route");
-        publish_route(&user, &target)?
-    } else {
+    Ok(())
+}
+
+fn deploy(user: &GlobalUser, target: &Target) -> Result<(), failure::Error> {
+    let routes = target.routes()?;
+    log::info!("routes: {:#?}", &routes);
+
+    if routes.is_empty() {
```

Comment:
i'm nearly inclined to move this all just into the top level command function.

Reply:
i think i'd rather it be its own function than in the top level

Reply:
i guess what i meant was, all the std out output and switching on routes etc. but i also think i might futz with this so i'm into it staying here